backend/test_appdata/test-fluid_test_copy_request_as_curl.json

```diff
@@ -3,13 +3,13 @@
   [
     "SetHandler",
     91390945,
-    { "x": 576, "y": 32 },
+    { "x": 76, "y": 32 },
```

Comment:
with the original coordinates, the relevant UI was hidden under the "first time" prompts. Is this a reasonable fix?

Reply:
The first time prompts aren't supposed to be there. We write some values to local storage to ensure they aren't so it sounds like something is broken there.

Reply:
My issue here must have been related to something else - test is passing now with original position.